src/groundlight/internalapi.py

```diff
@@ -255,17 +255,23 @@ def submit_image_query_with_inspection(  # noqa: PLR0913 # pylint: disable=too-m
         inspection_id: str,
         patience_time: Optional[float] = None,
         human_review: str = "DEFAULT",
+        metadata: Optional[dict] = None,
+        want_async: Optional[bool] = False,
```

Comment:
We need to add documentation for these parameters for them to be added to the docs!

Reply:
Check this out: https://github.com/groundlight/python-sdk/blob/4369348830180587faf4ff0ffd397adf69b2237e/src/groundlight/client.py#L241

Reply:
@blaise-muhirwa , the line you highlighted is for a different method, get_or_create_detector. 

Do I need to add any documentation for this PR? I only made changes to internalapi.py, which generally doesn't have much documentation.